qiskit_experiments/library/calibration/fine_amplitude.py

```diff
@@ -241,7 +245,7 @@ def circuits(self, backend: Optional[Backend] = None) -> List[QuantumCircuit]:
 
             circuit.metadata = {
                 "experiment_type": self._type,
-                "qubits": (self.physical_qubits[0],),
+                "qubits": self.physical_qubits,
```

Comment:
Just curious -- what led to this change?

Reply:
Style consistency rather than anything else. Some experiments are using `self.physical_qubits` (like RamseyXY and Drag) while others do `(self.physical_qubits[0], )` since the property `physical_qubits` returns `Tuple[int]` it seems more natural to use it as is rather than deconstruct it to reassemble it.